src/components/list_group/list_group_item.test.tsx

```diff
@@ -223,6 +223,27 @@ describe('EuiListGroupItem', () => {
         expect(container.firstChild).toMatchSnapshot();
       });
     });
+
+    describe('toolTipProps', () => {
+      test('renders custom tooltip props', async () => {
+        const { getByTestSubject } = render(
+          <EuiListGroupItem
+            label="Label"
+            toolTipText="Tooltip"
+            showToolTip
+            data-test-subj="trigger"
+            toolTipProps={{
+              anchorProps: {
+                'data-test-subj': 'tooltip-anchor',
+              },
```

Comment:
There's no reason to `waitForEuiToolTipVisible` if we're not testing the actual tooltip itself:

```suggestion
              'data-test-subj': 'tooltip',
```

Reply:
^ This is my last change request for this PR, and it's good to merge after!